godel_scan_analysis/src/keyence_scan_server.cpp

```diff
@@ -0,0 +1,79 @@
+#include "godel_scan_analysis/keyence_scan_server.h"
+
+#include <pcl_ros/transforms.h>
+#include <pcl/filters/voxel_grid.h>
+
+/*
+*/
+
+godel_scan_analysis::ScanServer::ScanServer(const std::string& world_frame, 
+                                            const std::string& scan_frame)
+  : map_(new ColorCloud)
```

Comment:
Apparently there is and I had no idea, so thanks for pointing it out. In this case I think I'm safe though because ColorCloud defines a constructor is neither a POD nor a trivial class:

See [this stackoverflow question](http://stackoverflow.com/questions/620137/do-the-parentheses-after-the-type-name-make-a-difference-with-new).